microservice/challenge-4/mood-consumer/src/main/java/ca/antaki/www/cat/mood/consumer/listener/CatMood.java

```diff
@@ -1,8 +1,24 @@
 package ca.antaki.www.cat.mood.consumer.listener;
 
-//Usually i dont put variables public but for speed
 public final class CatMood {
-	public String name;
-	public byte mood;
-	public long time;
```

Comment:
why not making them public final ?

Reply:
This was my first commit i since changed it to private final fields with getter/setters